src/dev/flang/tools/docs/Util.java

```diff
@@ -89,17 +89,17 @@ static String commentOf(AbstractFeature af)
     var commentsOfRedefinedFeatures = af
       .redefines()
       .stream()
-      .map(f -> System.lineSeparator() + "redefines " + f.qualifiedName() + ":" + System.lineSeparator()
+      .map(f -> "<br />" + "redefines " + f.qualifiedName() + ":" + "<br />"
         + commentOf(f))
       .collect(Collectors.joining(System.lineSeparator()));
```

Comment:
Does this newline have to be replace by <br> as well?

Reply:
I think it's probably better to not use newlines/br at all and put the redefined features comments in divs with a class like fd-inherited-comment or something.